firebase-database/src/test/java/com/google/firebase/database/util/ComparatorTester.java

```diff
@@ -165,11 +164,10 @@ private final void doTestEqualsCompatibility() {
       for (Object reference : referenceGroup) {
         for (List<Object> otherGroup : equalityGroups) {
           for (Object other : otherGroup) {
-            assertWithMessage(
-                    "Testing equals() for compatibility with compare()/compareTo(), "
-                        + "add a call to doNotRequireEqualsCompatibility() if this is not required")
+            // Testing equals() for compatibility with compare()/compareTo(),
```

Comment:
Per above, I had to use only assertWithMessage, not `.named` I kept the printf text formerly in `.named` and moved the other message to a comment to avoid losing information.